src/App.tsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useState, useEffect } from 'react';
+import { HashRouter as Router } from 'react-router-dom';
+
+import { AddTodoForm } from './components/AddTodoForm';
+import { TodoList } from './components/TodoList';
+import { TodosFilter } from './components/TodosFilter';
+import { Todo } from './types/types';
+
+export const App: React.FC = () => {
+  const [inputText, setInputText] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState('all');
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+
+  const filterHandler = () => {
+    switch (status) {
+      case 'completed':
+        setFilteredTodos(todos.filter(todo => todo.completed));
+        break;
+      case 'active':
+        setFilteredTodos(todos.filter(todo => !todo.completed));
+        break;
+      default:
+        setFilteredTodos(todos);
+    }
+  };
+
+  const saveLocalTodos = () => {
+    localStorage.setItem('todos', JSON.stringify(todos));
+  };
+
+  const getLocalTodos = () => {
+    if (localStorage.getItem('todos') === null) {
+      localStorage.setItem('todos', JSON.stringify([]));
+    } else {
+      setTodos(JSON.parse(localStorage.getItem('todos') || ''));
+    }
+  };
+
+  const countCompleted = () => {
+    let quantityCompleted = 0;
+
+    todos.forEach(todo => {
+      if (todo.completed) {
+        quantityCompleted += 1;
+      }
+    });
+
+    return todos.length - quantityCompleted;
```

Comment:
You just can make filter with completed todo  and find new array length

Reply:
I guess it would make the same effect - 1 itteration of array